mon-pix/app/components/update-expired-password-form.js

```diff
@@ -21,21 +22,13 @@ const VALIDATION_MAP = {
   },
 };
 
-const SUBMISSION_MAP = {
-  default: {
-    status: 'default', message: null,
-  },
-  error: {
-    status: 'error', message: ERROR_PASSWORD_MESSAGE,
-  },
-};
-
 export default class UpdateExpiredPasswordForm extends Component {
+
   @service intl;
   @service session;
   @service url;
 
-  @tracked validation = VALIDATION_MAP['default'];
+  @tracked validation = VALIDATION_MAP.default;
   @tracked newPassword = null;
   @tracked isLoading = false;
   @tracked authenticationHasFailed = null;
```

Comment:
Déclarer la variable errorMessage 
```suggestion
    @tracked errorMessage = null;
```

Reply:
Si la variable errorMessage n'est pas réinitialiser, on se retrouve avec les deux erreurs affichées.
[image](https://user-images.githubusercontent.com/10045497/95569158-7d77c100-0a25-11eb-9bea-67e87f52aaa5.png)

Reply:
Done